services/elections.request.service.ts

```diff
@@ -0,0 +1,340 @@
+
+@Injectable()
+export class ElectionsRequestService extends RequestService {
+  baseURL = 'elections';
+
+  constructor(http: HttpClient) {
+    super(http);
+  }
+  ////////////////////
+  // Elections
+  ///////////////////
+  /**
+   * Lists elections
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/get_election
+   * @param queryParams
+   * @returns an observable of all elections or elections specified by urlParams
+   */
+  listElection(queryParams?: any): Observable<Election[]> {
+    const electionsObservable = super.get(`${this.baseURL}/election`, queryParams).pipe(
+      map((data: {elections: Election[]}) => data.elections)
+    );
+    return electionsObservable;
+  }
+
+  /**
+   * Read election
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/get_election__election_id_
+   * @param queryParams
+   * @return a single election JSON object
+   */
+  readElection(queryParams: any): Observable<Election> {
+    const electionObservable = super.get(`${this.baseURL}/election/${queryParams}`);
+    return electionObservable;
+  }
+
+  /**
+   * Read current election
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/get_current
+   * @returns a single election JSON object
+   */
+  readElectionCurrent(): Observable<Election> {
+    const electionObservable = super.get(`${this.baseURL}/current`);
+    return electionObservable;
+  }
+
+  /**
+   * Create elections
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/post_election
+   * @param data
+   * @returns JSON object containing the election info created
+   */
+  createElection(data: any): Observable<Election> {
+    const electionsObservable = super.post(`${this.baseURL}/election`, data);
+    return electionsObservable;
+  }
+
+  /**
+   * Update election
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/put_election__election_id_
+   * @param data
+   * @param queryParams
+   * @returns JSON object containing the elction info that was updated
+   */
+  updateElection(data: any, queryParams: any): Observable<Election> {
+    const electionObservable = super.put(`${this.baseURL}/election/` + queryParams, data);
+    return electionObservable;
+  }
+
+  /**
+   * Count election votes
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/get_election__election_id__count
+   * @param queryParams
+   * @return JSON object of votes for candidates
+   */
+  readElectionCount(queryParams: any) {
```

Comment:
If we have an Election Count type we should use that, otherwise we should add that as an interface at some point.